README.md

```diff
@@ -199,10 +199,13 @@ ssha is not an SSH client; it instead figures out the right `ssh` command to run
 ```js
 ssh {
   /*
-  The default username can be overridden if necessary.
+  The default username/prefix can be overridden if necessary.
   If not defined, the default SSH user name is used.
   */
-  username = "ec2-user"
+  user {
```

Comment:
I think this is a bad example. You'd only want to set `ssh.user.name` OR `ssh.user.prefix` but not both at the same time (even though it would technically work). I'm also not sure about renaming `ssh.username` to `ssh.user.name` as that will break existing configs. How about keeping `ssh.username` as it was, and adding `ssh.username_prefix`? And in the README have it as a separate bit with its own comment so it looks slightly less like they're being used at the same time.